script/1_0_2current_en_script/Arcueid/Day 3/03_00_ARC03_3.txt

```diff
@@ -93,52 +93,52 @@ For some reason, Senpai smiles happily.
 "?"
 Arihiko and I look at each other and tilt our heads.
 <Page29>
-"I envy you. I admire friends who can understand each other like that without any hard feelings."
+"I really envy you. I wish I had friends who could understand each other without all the fervor."//「羨ましいなあ。そうやって何の気負いもなしで分かり合える友人がいるのって、憧れます」"really" added to make it more casual
 Senpai breathes out in approval.
 ""Is that so?""
 Arihiko and I look at each other, furrowing our brows.
 <Page30>
-"Well, it seems it's about time. I'm going back, but Tohno-kun, did you see the news this morning?"
-"Oh, ...I've been so busy I forgot. I haven't seen it yet."
+"Well, it seems it's about that time. I'm going back, but before I do, Tohno-kun, did you see the news this morning?"//「と、そろそろ時間ですね。わたしは戻りますけど、遠野くん、今朝のニュース見ましたか？」
```

Comment:
Don't really like the added 'that'.

'It seems it's almost time' maybe?

Reply:
I think `about that time` is ok, it's definitely an expression I use